bridges/GelbooruBridge.php

```diff
@@ -76,6 +76,10 @@ public function collectData(){
 			$posts = $posts->post;
 		}
 
+		if (is_null($posts)) {
+			returnServerError('No posts found.');
```

Comment:
Do we want this? It generates `HTTP 500 Internal Server Error` from us. Maybe better to give user an empty feed?

Reply:
Due to how the API works, this should only ever occur when someone is testing/creating a new feed via the web interface. Once they have a working feed URL, it should never generate this error. It may generate the exact same result every single time it's queried, if it's an unpopular tag, but the results should never shrink/disappear. Unless somehow all items with the queried tag are subsequently deleted from the site. If the the site itself goes down or the API changes/stops working, that should fail at a different point in the code.